### ISSUE TITLE
Explicitly display untracked browsers

### ISSUE THREAD

**Issue by Carpetsmoker** [Fyrd/caniuse#4772]
For the [CSS 2.1 selectors](https://caniuse.com/#feat=css-sel2) it lists "94.77% support" Yikes, what? It's not clear from the UI at all what the 5.33% that *don't* support this feature use.

I looked at [the datafile](https://github.com/Fyrd/caniuse/blob/master/features-json/css-sel2.json) and only IE 5.5 and 6 have a value other than "y", so it's actually supported by practically every user(?)

I then discovered that "all users" is a dropdown, and changing it to "all tracked" made the figure jump to "99.99%", which seems to match what's in the data file. 

I can't readily find what an "untracked" browser is, but I presume this is a browser that caniuse doesn't list in the feature table? e.g. ObscureBrowser 2.9 or whatnot. Basically it's "unknown".

Encountered issues:

1. The big percentage number displayed on initial load is incomplete at best, and misleading at worst.

2. It's unclear what a "tracked" browser/user is.

3. It's unclear that "all users" is an UI element. Restoring the native dropdown control by removing `-webkit-appearance: none` makes this a lot clearer (I'm using Firefox, which seems to pick up on this rule, too.

In #4058 it was proposed to change the default value to "all tracked", which seems to be soft-rejected with:

> might lead to situations where some new browser starts appearing that's not tracked yet and people will miss the fact that a feature may not be as well supported

Which seems reasonable enough.

So my proposal is to explicitely display it by default. Some mockups that I made by frobbing with the web inspector:

![2019-02-15-155335_654x168_scrot](https://user-images.githubusercontent.com/1032692/52831644-4363f800-313a-11e9-8970-85bb248ef268.png)

![2019-02-15-155535_707x110_scrot](https://user-images.githubusercontent.com/1032692/52831645-43fc8e80-313a-11e9-939a-cd8131127cbc.png)

In the second example the blue text would have "Untracked browsers" as the title attribute.

This will only solve the first of the three issues, but that's probably by far the bigest one. The second one can be solved by adding a more expansive help text to the title attribute, or a link to a help page. The third one goes away as this UI element is no longer that critical.

What do you think? I'll be happy to implement this, just wanted to discus first.

Thanks!

**Comment by atomiks** [comment]
Yes this makes sense. The current value is pretty misleading - I remember back in 2017 when "Flexbox" showed 97.5% support, now it's down to 94.4%. All Tracked is at 99.6%.

The current UI makes it seem like virtually every feature is unusable by at least 5% of the world

**Comment by Fyrd** [comment]
Some good points made here, thanks. Will consider a more explicit way of displaying this amount. In the mean time I have fixed the select list UI for Firefox.

**Comment by anacoxta** [comment]
> It's unclear what a "tracked" browser/user is.

Maybe a small FAQ could help clear this up for programming newbies like myself :)
I'm checking the site out and I'm a little confused by the "all users" versus "all tracked" results too.

**Comment by Fyrd** [comment]
Alright, have added an info button next to the dropdown list that explains what the usage values mean. I'd like to avoid adding too much data by including the untracked % all the time (since there can also be unprefixed support & multiple usage sources in that same are) but you can now see that amount when hovering over the total %. 

There's also an issue where the untracked % should only be ~2.5% instead of ~5% so that's something else I'm looking into.

**Comment by Fyrd** [comment]
Okay, found the remaining issue, untracked % is now down to 1.63% so numbers are looking a lot better. Closing this issue.
